src/stories/apps/GroupChannelApp.stories.tsx

```diff
@@ -136,6 +137,12 @@ const meta: Meta<typeof App> = {
         'A property that determines whether to automatically select another channel when the currently selected channel is deleted, or the user exits the channel, causing it to be deselected in the channel list.',
       control: 'boolean',
     },
+    htmlTextDirection: {
```

Comment:
![Screenshot 2024-06-11 at 4 06 45 PM](https://github.com/sendbird/sendbird-uikit-react/assets/10060731/44407193-4efa-44a0-9e7a-f63dc463cda2)
Added this one too for the easier test :)